.travis.yml

```diff
@@ -11,8 +11,8 @@ addons:
 
 branches:
   only:
-  - master
-  - develop
+    - travis_build
```

Comment:
Hi @hoangviet1993, I think you need not change these...

Reply:
Whoops! I think these got merged when I merged from my own branch. I forgot to do the --assume-unchanged for .travis.yml when I re-enable learnerDashboard the first time. I checkout a new copy and made changes accordingly 👍